### DISCUSSION TITLE
Automatic insertion for contributing to dynamic-theme-fixes.config?

### DISCUSSION THREAD

**Discussion by KamilDev** [darkreader/darkreader#13664] [Q&A]
Just wondering if there exists an easier/automatic way to contribute fixes to `dynamic-theme-fixes.config` so I don't have to manually locate the alphabetic position to place my site fix in?

**Comment by Myshor** [comment]
There is not but...

If you add your part of fixes into developer tools of Dark Reader and click Apply button then it will be sorted in developer tools window.

With that you can again search for the URL with fixes and check where exactly it is located in sitefixes.

Example below:
![obraz](https://github.com/user-attachments/assets/96222f7f-0720-4d3d-8807-d53560c3890c)

Pro-Tip:
I always do that and check what will be the next URL after fix.

Then I can copy/paste:
```
fix.url.example

INVERT
.just-example-of-fix

================================

```
Put the cursor in git hub on the beginning of the found next URL and paste.
